### PR TITLE
feat: Update Navigation bar and footer

### PR DESCRIPTION
**Pull request by viru2001**
## Add labels in navigation items 

image

### CODE REVIEW

**Review by bhavyaj12 — APPROVED** [review]
Looks good to me, the footer font size can be smaller though

**Comment by viru2001** [comment]
> 

okay. will do it

File: index.html [hunk 1]
```diff
@@ -20,43 +17,65 @@
...
+          class="brand-name text-dec-none headline-lg font-wt-bold p-4"
+          href="#"
+          >Gamers Bazaar</a
+        >
 
         <div class="search-box d-flex align-center justify-between my-8 w-50">
           <input
             type="text"
-            class="search-input w-100 p-2 m-2 text-sm"
+            class="search-input p-2 m-2 text-sm"
```
Comment (irajbhinde) [inline index.html:29]: In future you can make class names more user friendly. padding & margin is clear to me but might not be for everyone looking at your website

File: pages/signup.html [hunk 2]
```diff
@@ -76,9 +94,7 @@
           </div>
 
           <div class="d-flex flex-col px-8 py-4">
-            <label class="text-sm my-2" for="email"
-              >Name</label
-            >
+            <label class="text-sm my-2" for="email">Name</label>
```
Comment (irajbhinde) [inline pages/signup.html:97]: for="email" whereas the label is being used for name

**Review by irajbhinde — COMMENTED** [review]
1) Checkout the P0 release criteria for ecom on lms, you've missed out on many stuffs. First Name, Last Name & hide / show password icon. 
2) It's good that you are using the component library at most of the places. Keep up the good work for that.

File: index.html (hunk 1 above)
Comment (viru2001) [reply index.html:29]: I will make documentation for utility classes in my component library

File: pages/signup.html (hunk 2 above)
Comment (viru2001) [reply pages/signup.html:97]: yes I will correct it now

**Comment by viru2001** [comment]
> 1. Checkout the P0 release criteria for ecom on lms, you've missed out on many stuffs. First Name, Last Name & hide / show password icon.
> 2. It's good that you are using the component library at most of the places. Keep up the good work for that.

Thank you for telling me this. I will add missing things

**Pull request merged**